pages/services/kubernetes/2.1.1-1.12.4/getting-started/setting-up/index.md

```diff
@@ -29,30 +29,30 @@ before continuing on.
 
 The Mesosphere Kubernetes Engine (MKE) requires access over HTTPS in order to connect to the kubernetes-apiserver using `kubectl`. So to finish setting up the cluster and before moving on to installing the MKE, let's go ahead and make sure you are authenticated to the DC/OS CLI using HTTPS:
 
-1. **In your CLI, enter:**
+1. <strong>In your CLI, enter</strong>:
```

Comment:
Again, were our repos out of sync @chrisgaun? We shouldn't have these types of docs changes on a package version release.